package.json

```diff
@@ -344,12 +344,13 @@
     "@brave/react-virtualized-auto-sizer": "^1.0.4",
     "@glif/filecoin-wallet-provider": "2.0.0-beta.12",
     "@ledgerhq/hw-app-eth": "6.24.1",
+    "@ledgerhq/hw-app-solana": "6.27.1",
```

Comment:
action item for myself is to review this still, but wanted to get the feedback to you that I found so far

Reply:
review left in the security issue [here](https://github.com/brave/security/issues/919#issuecomment-1161036996)